File: 2523/2523.cs
public class Solution {
    public int ClosestPrimeNumbersInRange(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(ClosestPrimeNumbersInRange(root.left), ClosestPrimeNumbersInRange(root.right));
    }
}

File: 2523/2523.go
func closestPrimeNumbersInRange(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := closestPrimeNumbersInRange(root.Left)
	r := closestPrimeNumbersInRange(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}